src/utils/fetch-utils.ts

```diff
@@ -7,14 +7,16 @@ interface Innhold {
 
 export const BASE_URL = '/arbeidsgiver-permittering';
 
-export const isProduction = (): string =>
-    process.env.NODE_ENV === 'production' ? '' : 'http://localhost:3001';
+export const isProduction = (): boolean =>
+    process.env.NODE_ENV === 'production';
```

Comment:
Tror vi har NODE_ENV=production både i dev og i prod, bare ikke lokalt? 🤔

Reply:
Ja production settes i Dockerfile så den vil være for både prod og dev. Tenkte litt på det, men er det egentlig så viktig for sentry? Det som logges fra dev kan jo være like viktig som det fra prod?

Men kan ta de ut i kode f eks. Secrets vil jo ikke virke da de må settes build time.

Reply:
Vi er ikke i GCP, så testmiljø kan bare nås via Chrome SKSS (Citrix). Det kommer en del støy i loggene her ifm at det er færre tjenester man kan bruke derfra. 
Men vi bruker jo ikke testmiljøet så veldig mye, så jeg er forsåvidt med på å prøve å ha dem i samme bucket for now